index.html

```diff
@@ -34,5 +34,77 @@ <h1 class="header-logo-wrapper">
         </div>
       </div>
     </section>
+    <section class="about">
+      <div class="about-wrapper">
+        <h2 class="about-title">
+          What is <span class="about-title-accent-1">Grand</span>
+          <span class="about-title-accent-2">Players</span>?
+        </h2>
+        <div class="about-text-and-image">
+          <p class="about-text">
+            Grand Players
+            は従量課金制のオンデマンド動画を用いたオンラインギターレッスンサービスです。好きな時間に、自分にあったレベルでギターの上達を目指すことができます。
+            また、複数のレッスンをまとめた「コース」を受講することで、より体系的にテクニックを身に着けることもできます。
+            「コース」を受講された方は、現役ギタリスト講師との個別面談で、より最適なレッスンの選択をサポートいたします。
+          </p>
+          <img class="about-image" src="img/live.png" alt="ライブ風景" />
+        </div>
+      </div>
+    </section>
+    <section class="courses">
+      <div class="course-wrapper">
+        <h2 class="course-title">Courses</h2>
+        <div class="course-cards">
+          <a href="" class="course-card">
+            <div class="course-card-title-img">
+              <h2 class="course-card-title">Frontman</h2>
+            </div>
```

Comment:
```suggestion
        <h2 class="course-title">Courses</h2>
        <div class="course-cards">
          <a href="" class="course-card">
            <div class="course-card-title-img">
              <h3 class="course-card-title">Frontman</h3>
            </div>
```
細かいところですが、階層構造としてはこちらが適切かと思います。